proto/vtorcdata.proto

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2025 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This file contains the Vitess topology related data structures.
+// Very few of these structures are exchanged over the wire (only
+// TabletType and KeyRange), but they are all used by the topology
+// service.
+
+syntax = "proto3";
+option go_package = "vitess.io/vitess/go/vt/proto/vtorcdata";
+
+option java_package="io.vitess.proto";
+
+package vtorcdata;
+
+// KeyspaceConfig stores configuration for Vtorc.
+message KeyspaceConfig {
+  // DisableEmergencyReparent reflects if EmergencyReparentShard
+  // can be used in Vtorc recoveries.
+  bool disable_emergency_reparent = 1;
+}
+
+// ShardConfig stores configuration for Vtorc.
+message ShardConfig {
```

Comment:
`ShardConfig` == `KeyspaceConfig` currently, but I plan for the two to diverge in future PRs